src/ext-mpxy.adoc

```diff
@@ -655,19 +653,24 @@ sbi_mpxy_send_message_without_response(uint32_t channel_id,
                                        uint32_t message_id,
                                        unsigned long message_data_len)
 ```
-
-Send a message to the MPXY channel specified by the `channel_id` parameter.
-The `message_id` parameter specifies a message specific to a message protocol
+Send a message to the MPXY channel specified by the `channel_id` parameter
+without waiting for a message response from the MPXY channel. The `message_id`
+parameter specifies the message protocol specific identification of the message
 to be sent whereas the `message_data_len` parameter represents the length of
 message data in bytes which is located at the offset `0x0` in the shared memory
 setup by the calling hart.
 
-This function does not wait for response and returns after successful message
-transmission.
+This function does not wait for message response from the channel and returns
+after successful message transmission. In cases where complete data transfer
+requires multiple transmissions, the supervisor software shall send multiple
+messages as necessary. Details of such cases can be found in the respective
+message protocol specification.
 
-In cases where complete data transfer requires multiple transmissions, the
-supervisor software shall issue multiple requests as necessary. Details of such
-cases can be found in the respective message protocol specification.
+NOTE: The messages which do not have an expected response as per the underlying
+message protocol specification are also referred to as posted messages. This
+function should be only used for such posted messages and the respective message
+protocol specification should define a mechanism to track the status of posted
+messages using notification events or some other message with response.
```

Comment:
The 2nd part is optional - Correct ? Can we clarify that ? Something like this:
This function should be only used for such posted messages. The respective message
protocol specification should define a mechanism to track the status of posted
messages using notification events or some other message with response if required.

Reply:
Okay, I will update.